package.json

```diff
@@ -35,9 +24,12 @@
     "node": ">=4.0.0"
   },
   "scripts": {
-    "test": "jest",
-    "test:lint": "tslint 'test-lint/**'",
-    "test:prod": "npm t && npm run test:lint",
+    "prepublish": "npm run build-tools && npm run generate-config",
+    "lint": "tslint 'tools/**'",
+    "test": "npm run clean && jest --no-cache",
+    "clean": "rimraf ./node_modules/vrsource-tslint-rules/rules/*.ts",
```

Comment:
What is this for? Why do we need to remove something inside node_modules?

Reply:
It's an tslint issue, tslint can't load rules that their *.ts and *.js exist simultaneously (while using `rulesDirectory`), unfortunately, `vrsource-tslint-rules` is the only ruleset that publish *.ts with *.js.

![image](https://user-images.githubusercontent.com/8341033/29762200-ce4f4622-8bff-11e7-83fc-ac4b679f3232.png)

Reply:
Interesting :D good catch